catalog/spatial/registry/registry-identification-plugin/src/test/java/org/codice/ddf/registry/identification/IdentificationPluginTest.java

```diff
@@ -72,6 +76,8 @@
 
   private MetacardImpl sampleData;
 
+  @Mock private RegistryIdRetriever registryIdRetriever;
```

Comment:
Is this used anywhere outside of the `setUp()` method? Can it be just defined in that method instead?
```java
    final RegistryIdRetriever registryIdRetriever = mock(RegistryIdRetriever.class);
    when(registryIdRetriever.getRegistryIdInfo())
        .thenReturn(new RegistryIdRetriever.RegistryIdInfo(Collections.emptyList()));
    identificationPlugin.setRegistryIdRetriever(registryIdRetriever);
```